<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyATK i.MX Bootstrap Kit</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <attribute name="title">
        <string>Bootstrap Application</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Bootstrap Memory Browser</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Peek/Poke Device Memory</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="2" colspan="2">
            <widget class="QLineEdit" name="mbrowser_pp_write_value"/>
           </item>
           <item row="2" column="0" colspan="4">
            <widget class="QTableView" name="mbrowser_pp_table"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Data size</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="mbrowser_pp_datasize"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Memory address</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="mbrowser_pp_address">
             <property name="text">
              <string>0x00000000</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>P&amp;eek</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>P&amp;oke</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Write value</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="4">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Device Memory Browser</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="4" column="1" colspan="2">
               <widget class="QPushButton" name="mbrowser_browse_save_button">
                <property name="text">
                 <string>&amp;Save to file...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="mbrowser_browse_datasize"/>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Data size</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2" colspan="2">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Read length</string>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QPushButton" name="mbrowser_browse_dump_button">
                <property name="text">
                 <string>&amp;Dump memory</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2" colspan="2">
               <widget class="QSpinBox" name="mbrowser_browse_readlength">
                <property name="suffix">
                 <string> bytes</string>
                </property>
                <property name="maximum">
                 <number>128</number>
                </property>
                <property name="value">
                 <number>128</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Memory address</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="mbrowser_browse_address">
                <property name="text">
                 <string>0x00000000</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0" colspan="4">
               <widget class="QTableView" name="mbrowser_browse_dump_table"/>
              </item>
             </layout>
             <zorder>mbrowser_browse_dump_table</zorder>
             <zorder>mbrowser_browse_datasize</zorder>
             <zorder>label_9</zorder>
             <zorder>mbrowser_browse_address</zorder>
             <zorder>label_10</zorder>
             <zorder>mbrowser_browse_dump_button</zorder>
             <zorder>label_13</zorder>
             <zorder>mbrowser_browse_readlength</zorder>
             <zorder>mbrowser_browse_save_button</zorder>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Flash Tool</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Serial Terminal</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open_binary_file"/>
    <addaction name="actionView_log"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Application Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTreeView" name="app_browser_tree"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Board Selection</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Device Connection</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QComboBox" name="dev_select_combo"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Board Support Package (BSP) Browser</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListView" name="dev_select_bsp_list"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open_binary_file">
   <property name="text">
    <string>&amp;Open binary file...</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionView_log">
   <property name="text">
    <string>View &amp;log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
